openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -393,11 +393,12 @@ The response from the Credential Issuer that contains a Credential Offer Object
 
 This ability to pass the Credential Offer by reference is particularly useful for large requests.
 
-Below is a non-normative example of the Credential Offer displayed by the Credential Issuer as a QR code when the Credential Offer is passed by reference:
+When the Credential Offer is passed by reference, the Credential Offer can be displayed by the Credential Issuer as a QR code. Below is a non-normative example:
 
 ```
 openid-credential-offer://?
   credential_offer_uri=https%3A%2F%2Fserver%2Eexample%2Ecom%2Fcredential-offer.json
+  %2FGkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM
```

Comment:
```suggestion
  credential_offer_uri=https%3A%2F%2Fserver%2Eexample%2Ecom%2Fcredential-offer
  %2FGkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM
```
I would remove the .json if we want to show a random uri:
`https://server.example.com/credential-offer/GkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM` is a more natural URI compared to 
`https://server.example.com/credential-offer.json/GkurKxf5T0Y-mnPFCHqWOMiZi4VS138cQO_V7PZHAdM` imho

Reply:
i felt the same, but checked RAR examples and they had .jwt examples, so I kept it.